src/components/file-component/FileComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react'
+
+import { format } from 'date-fns'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum, File } from '~/types'
+import { styles } from '~/components/file-component/FileComponent.styles'
+
+interface FileComponentProps {
+  file: File
+}
+
+const formatDate = (date: Date) => {
```

Comment:
try to use our formattedDate function for this

Reply:
> 

our `getFormatedDate(date)` is returning date in next format: Month Day, Year
![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/ee97d60c-2fd8-4841-922f-a79cb4a74e03)

but `format(date, 'formating-params')` from `date-fns` returning it in format like it's in figma: Day(with an ending) Month Year
![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/8f77cf56-1054-4431-a1e5-f2d90eebe644)

Figma preview:
![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/65914316-1201-4050-ac74-6ad5e829955d)

should i use our `getDormatedDate()` anyway?

Reply:
I think it is better to use getFormatedDate, there are no critical differences

Reply:
> I think it is better to use getFormatedDate, there are no critical differences

fixed

Reply:
move this function inside component and it can look like:
Also, I think include ordinal is redundant, it is not an important difference between "3th Jully 2023" and "Jully 3 2023"

const formattedDate = getFormatedDate({
    date: file.uploadedDate,
    options: {
      day: 'numeric',
      month: 'long',
      year: 'numeric'
    }  })

Reply:
> move this function inside component and it can look like: Also, I think include ordinal is redundant, it is not an important difference between "3th Jully 2023" and "Jully 3 2023"
> 
> const formattedDate = getFormatedDate({ date: file.uploadedDate, options: { day: 'numeric', month: 'long', year: 'numeric' } })

fixed it 👍